asynctnt/connection.py

```diff
@@ -260,13 +344,14 @@ async def full_connect():
                                 conn = self._loop.create_unix_connection(
                                     functools.partial(self.protocol_factory,
                                                       connected_fut),
-                                    unix_path
-                                )
+                                    unix_path,
+                                    ssl=ssl_context)
```

Comment:
maybe it will be good to allow specifying ssl_handshake_timeout as well (forward from Connection `__init__` params, with the default to None). In both `create_unix_connection` and `create_connection` calls.

Reply:
Added

Reply:
Timeout is supported only since 3.7. I've added a description note, but should I prove something more noticeable?